core/src/main/java/org/geysermc/geyser/session/GeyserSession.java

```diff
@@ -1425,11 +1425,28 @@ public boolean isConsole() {
         return false;
     }
 
+    @Override
+    public @NonNull UUID playerUuid() {
+        return playerEntity.getUuid();
+    }
```

Comment:
same as above, this could cause an NPE if it's called when we don't have a player entity yet (e.g. session login event)

Reply:
I could revert these changes in favour of your PR, but `playerUuid()` is a new method. 

I may as well implement it correctly (what you did)... and at that point I may as well implement the others correctly as well.

thoughts?